libvirt/tunnel/tunnel.sh

```diff
@@ -1,98 +1,163 @@
 #!/bin/bash
+# Load profile vars
+. profile_s390x.sh
```

Comment:
This will always be overwritten by the second call to source.

This line should be:
`. profile.sh`

Also, we need seperate profiles for profile_s390x_1 and profile_s390x_2, since the LPARs will have different cluster_id values. We should probably make cluster_id a constant, and just add it to profile.